pkgs/applications/graphics/graphicsmagick/default.nix

```
@@ -38,6 +73,13 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     sed -i 's/-ltiff.*'\'/\'/ $out/bin/*
+
+    # Install perl bindings
+    cd PerlMagick
+    perl Makefile.PL PREFIX=$out INSTALLDIRS=site PERL=$(type -P perl) FULLPERL=\"$fullperl/bin/perl\
```

For that perl also need to go to nativeBuildInputs.

```suggestion
    perl Makefile.PL PREFIX=$out INSTALLDIRS=site PERL=${perl}/bin/perl FULLPERL=\"$fullperl/bin/perl\
```